### PR TITLE
Simplify table rows, allow middle click 

### PR DESCRIPTION
**Pull request by mvaivre**
Tables structure has been changed quite a bit. 
Need some focus on hands-on testing.

(I would have loved to clean up the styling which is a bit all over the place - between Table, TableRow etc.), but this would take too much time & effort, and it works as it is now. Will do next year ;))

### CODE REVIEW

File: src/components/Table/TableRow.tsx [hunk 1]
```diff
@@ -31,14 +33,42 @@ const rowVariants = {
   shown: { opacity: 1 }
 }
 
-const TableRow: FC<RowProps> = ({ children, onClick, className }) => (
-  <motion.tr variants={rowVariants} transition={{ duration: 0.8 }} onClick={onClick} className={className}>
-    {children}
+const TableRow: FC<RowProps> = ({ children, onClick, linkTo, className }) => (
+  <motion.tr variants={rowVariants} transition={{ duration: 0.8 }} onMouseUp={onClick} className={className}>
+    {Children.map(children, (c) =>
```
Comment (nop33) [inline src/components/Table/TableRow.tsx:38]: Ha, I wasn't aware of the `Children` API! Looking at the docs I see that this API belongs to legacy and they recommend using alternatives: https://beta.reactjs.org/apis/react/Children

Maybe it's worth having a second look at this implementation. WDYT?

With a quick look, maybe something similar to how `headerTitles` is handled by the `TableHeader`?

File: src/pages/BlockInfoPage.tsx [hunk 2]
```diff
@@ -209,28 +207,21 @@ const TransactionRow: FC<TransactionRowProps> = ({ transaction }) => {
...
-          <Badge
-            type="neutralHighlight"
-            amount={outputs?.reduce<bigint>((acc, o) => acc + BigInt(o.attoAlphAmount), BigInt(0))}
-            floatRight
-          />
-        </td>
+        </span>
+        <Badge
+          type="neutralHighlight"
+          amount={outputs?.reduce<bigint>((acc, o) => acc + BigInt(o.attoAlphAmount), BigInt(0))}
```
Comment (nop33) [inline src/pages/BlockInfoPage.tsx:221]: nit: I know this line was not created in this PR, but... 😅 I think the render function would look much nicer if this reducer was moved into a `totalAmount` variable.

File: src/pages/TransactionInfoPage.tsx [hunk 3]
```diff
@@ -201,55 +195,47 @@ const TransactionInfoPage = () => {
...
-                    )}
-                  />
-                </td>
+                <b>Total Value</b>
+                <Badge
+                  type="neutralHighlight"
+                  amount={outputs?.reduce<bigint>(
+                    (acc, o) => acc + BigInt((o as Output).attoAlphAmount ?? (o as AssetOutput).attoAlphAmount),
+                    BigInt(0)
+                  )}
```
Comment (nop33) [inline src/pages/TransactionInfoPage.tsx:237]: nit: 🤐 😅

**Review by nop33 — COMMENTED** [review]
Wow, this whole change of table stuff to span stuff must have not been so much fun! Code review done, I'll do some functional testing now. 👏

**Comment by nop33** [comment]
Functional tests look good, except only the press on the down/up arrow in both the Block and Address pages (both when opened or closed)


https://user-images.githubusercontent.com/1579899/207356410-88757799-d8c4-4c5b-a5f6-3432b75a6db6.mp4

File: src/components/Table/TableRow.tsx (hunk 1 above)
Comment (mvaivre) [reply src/components/Table/TableRow.tsx:38]: Crap. I'll take a second look :)

Comment (mvaivre) [reply src/components/Table/TableRow.tsx:38]: TODO added

File: src/pages/BlockInfoPage.tsx (hunk 2 above)
Comment (mvaivre) [reply src/pages/BlockInfoPage.tsx]: Done

**Comment by mvaivre** [comment]
Everything's fixed + todo added :)

**Review by nop33 — APPROVED** [review]

**Pull request merged**